go/coinstacks/thorchain/daemon/readiness.sh

```diff
@@ -20,9 +20,9 @@ CATCHING_UP=$(echo $STATUS | jq -r '.result.sync_info.catching_up')
 NUM_PEERS=$(echo $NET_INFO | jq -r '.result.n_peers')
 
 status_curls=(
-  "curl -sf -m 3 https://rpc.ninerealms.com/status"
+  "curl -sf -m 3 https://rpc-v2.ninerealms.com/status"
   # referer header now required to avoid being blocked
-  "curl -sf -m 3 -H \"Referer: https://app.thorswap.finance\" https://rpc.thorswap.net/status"
+  "curl -sf -m 3 -H \"Referer: https://app.thorswap.finance\" https://rpc-v2.thorswap.net/status"
```

Comment:
confirm thorswap v2 endpoint at time of fork. option to comment out if unavailable initially

Reply:
https://github.com/shapeshift/unchained/pull/1016/commits/6168ba108c3680cc25d6dd1d8cd842664db47f47